### PR TITLE
Hp algs

### PR DESCRIPTION
**Pull request by Sait4ma**

### CODE REVIEW

File: src/NosCore.Algorithm/DamageService/DamageMinMaxService.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
+﻿using NosCore.Algorithm.HpService;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace NosCore.Algorithm.DamageService
+{
+    // Work in progress
+    public class DamageMinMaxService : IDamageMinMaxService
```
Comment (0Lucifer0) [inline src/NosCore.Algorithm/DamageService/DamageMinMaxService.cs]: I think DamageService and IDamageService are better name

**Comment by 0Lucifer0** [comment]
Don’t forget to add a test for damage and to run the test for the hp

**Comment by 0Lucifer0** [comment]
Also avoid to do a merge commit a rebase is a lot better

File: src/NosCore.Algorithm/DamageService/DamageMinMaxService.cs (hunk 1 above)
Comment (Sait4ma) [reply src/NosCore.Algorithm/DamageService/DamageMinMaxService.cs]: I use DamageMinMax cause i want to add on the same folder the HitRate, and the secondary stat

Comment (0Lucifer0) [reply src/NosCore.Algorithm/DamageService/DamageMinMaxService.cs]: Don’t use the same folder use a different service

**Pull request merged**